tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -127,3 +127,7 @@ def print(self,
     def _print(self, result: Mapping[str, Any]) -> Any:
         assert result["data"] is not None
         return result["data"]
+
+    @command
+    def set_viewport(self, context: str, viewport: Optional[Mapping[str, Any]] = None) -> Mapping[str, Any]:
+        return {"context": context, "viewport": viewport}
```

Comment:
Did you consider against using a `Viewport` class given that it is not used here, or haven't you updated that part yet?

Reply:
I am still trying to understand the benefits of using a class: since we want to test that the implementation reacts properly for any input from the client, it seems that using a class similar to SerializationOptions would restrict that as it does not allow sending None values, for example. So it looks like the current version allows testing any input. Or have I misunderstood how the proposed class would be used?

Reply:
Oh, good point. So yeah, looks like that this is indeed the most flexible approach and that we may have to update the script module as well to offer more flexibility for the `SerializationOptions`. As such lets keep what we have here now.